src/bundle/DependencyInjection/Configuration.php

```diff
@@ -168,8 +168,16 @@ private function addCustomStylesSection(NodeBuilder $ezRichTextNode)
      * ezpublish:
      *     ezrichtext:
      *         alloy_editor:
+     *             extra_plugins: [plugin1, plugin2]
+     *             extra_buttons:
+     *                 - paragraph: [button1, button2]
```

Comment:
I think the hyphens should be removed, it doesn't work with hyphens. :)

Reply:
@micszo nice catch! Fixed in https://github.com/ezsystems/ezplatform-richtext/pull/33/commits/00a8a69ac3431432f9ccb99cf6a6a6e38236206c